Sources/Testing/Expectations/ExpectationChecking+Macro.swift

```diff
@@ -108,17 +108,17 @@ public func __checkValue(
 /// - Throws: Whatever is thrown by `op`.
 private func _callBinaryOperator<T, U, R>(
   _ lhs: T,
-  _ op: (T, () -> U) throws -> R,
```

Comment:
We don't expand expressions containing `try` (currently), so this can't ever be throwing in practice.